rdesktop.c

```diff
@@ -697,6 +700,16 @@ main(int argc, char *argv[])
 					p++;
 				}
 
+				if (*p == '@')
+				{
+					g_dpi = strtol(p + 1, &p, 10);
```

Comment:
Ok, I found something scary of relevance:

  https://bugs.launchpad.net/ubuntu/+source/xorg-server/+bug/589485